src/definitions/modules/dropdown.less

```diff
@@ -214,6 +219,18 @@
   color: @itemDescriptionColor;
 }
 
+.ui.dropdown .menu > .item.vertical > .description {
+  margin: 0;
+}
+
+/*-----------------
+      Item Text
+-------------------*/
+
+.ui.dropdown .menu > .item.vertical > .text {
+  margin-bottom: 0.5em;
```

Comment:
Please create and use a new variable like `@verticaltemMargin: 0.5em` in https://github.com/fomantic/Fomantic-UI/blob/master/src/themes/default/modules/dropdown.variables

Reply:
I removed my additional request to adjust the menu template, because it can already be accomplished by providing `vertical item` to the field option `class`.

So only thing left is to turn the margin value into a variable. 🙂 
I also think 0.25em looks better as  @jamessampford  suggested

Reply:
@komiyama  Are you still interested in finishing this PR?

Reply:
@komiyama Once again, since it's half a year ago: Are you still interested in finishing this PR? Then please change the hardcoded 0.5em to a variable `verticalItemMargin` and use 0.25em as default as [stated above](https://github.com/fomantic/Fomantic-UI/pull/1116#discussion_r338178125).

Reply:
@lubber-de Yes, I can finish this PR.

Reply:
@rkomiyama After almost another 4 weeks 🙄 : Any estimations **when** you are going to finish this PR? 😉 The only thing left is turning the hardcoded 0.5em value into a 0.25em variable:

## Required changes
### 1. `src/definitions/modules/dropdown.less`:

Change from
```
.ui.dropdown .menu > .item.vertical > .text {
  margin-bottom: 0.5em;
}
```
to 
```
.ui.dropdown .menu > .item.vertical > .text {
  margin-bottom: @verticalItemMargin;
}
```

### 2. `src/themes/default/modules/dropdown.variables`:

Add 
```
/* Vertical Item */
@verticalItemMargin: 0.25em;
```
in line 66 just before the `/* Sub Menu */` variables are declared

We really like your PR and want to merge it. 🙂 

If you don't have time to do this yourself, please check the checkbox to the right which allow us to push to your branch  
![image](https://user-images.githubusercontent.com/18379884/87960360-7db29100-cab4-11ea-9451-b555f66b9663.png)

I just tries using gitpod, but i failed
![image](https://user-images.githubusercontent.com/18379884/87960443-9753d880-cab4-11ea-9b0d-0d4118c6dfa0.png)

#### Otherwise we are closing this PR and implement your changes on our own in a new PR.
Thanks

Reply:
Time flew by real quick. Okay, I'll prioritize this task before I do anything else today.

Reply:
I think my linter threw some things off. Let me go back and fix that.

Reply:
Phew, sorry about that. The PR should _finally_ be ready for review.

Reply:
Well, you have implement my suggested changes (it wasnt meant to remove anything else...), but...
Now you have also removed essential parts like the flexbox setting 😢 

Look at your own jsfiddle https://jsfiddle.net/hj824Ld3/

You need to reimplement the following again, otherwise your PR does not work anymore
```
.ui.dropdown .menu > .item.vertical {
  display: flex;
  flex-direction: column-reverse;
}

.ui.dropdown .menu > .item.vertical > .description {
  margin: 0;
}
```

as it was before  by  https://github.com/fomantic/Fomantic-UI/commit/c5f57d07016cc3f8d5f0a1e1b7e592b0aebdf0bb

Reply:
Big whoops. How about this?

Reply:
Now it's perfect. Thanks! 😄